src/wait.ts

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
Would be better to name it utils.ts or somethine like that

Reply:
Done